nncf/compression_method_api.py

```diff
@@ -157,9 +156,16 @@ def export_model(self,
             `save_path` - a path to the file for the exported model to be saved into.
             `input_names` - list of input tensors names (optional).
             `output_names` - list of output tensors names (optional).
-            *args, **kwargs - if the model's `forward` requires additional parameters
-            during export, specify these here.
+            `model_args` - tuple of additional positional and keyword arguments which are
+                required for the model's `forward` during export. Should be specified in
+                the following format:
+                    - (a, b, {'x': None, 'y': y}) for positional and keyword arguments
+                    - (a, b, {}) for positional arguments only
+                    - ({'x': None, 'y': y},) for keyword arguments only
```

Comment:
This probably can be expressed as type hints as well. Shouldn't remove this human-readable description, though

Reply:
I think it's impossible. All we can write is `Tuple[Any, ...]`.

Reply:
It might be better to use `model_args` like this
- `((a, b), {'x': None, 'y': y})` for positional and keyword arguments
- `((a, b), {})` for positional arguments only
- `((), {'x': None, 'y': y})` for keyword arguments only
This is more intuitive and in this case, we can specify the type hint `Tuple[Tuple[Any, ...], Dict[str, Any]]`.

Reply:
`Union[Tuple[Tuple[Any, ...], Dict[str, Any]], Tuple[Dict[str, Any]]`
We should exactly match the torch's way if we decided to be aligned with it.